### PR TITLE
Shawn/worksheet-9

### PR DESCRIPTION
**Pull request by coffee-intocode**

### CODE REVIEW

File: frontend/src/components/Main/Main.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        />
+        <Route exact path={routes.checkout} component={Checkout}></Route>
+        <Route exact path={routes.orders} component={Orders}></Route>
+        <Route
+          exact
+          path={routes.ordersSuccess}
+          component={OrderSuccess}
+        ></Route>
+        <Route exact path={routes.cart} component={Shoppingcart}></Route>
+      </Switch>
```
Comment (coffee-intocode) [inline frontend/src/components/Main/Main.js:44]: Routes added to frontend navigation.

File: frontend/src/utils/routes.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const routes = {
```
Comment (coffee-intocode) [inline frontend/src/utils/routes.js:1]: Routes export in utils for react app.

File: backend/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (ahaly92) [inline backend/.gitignore]: new line

File: backend/src/db/migrations/20200901213405-create-user.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.dropTable('Users');
+  }
+};
```
Comment (ahaly92) [inline backend/src/db/migrations/20200901213405-create-user.js]: new line

File: backend/src/db/models/user.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
```
Comment (ahaly92) [inline backend/src/db/models/user.js]: remove comments

File: backend/src/db/models/user.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  User.init({
+    username: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.STRING
+  }, {
+    sequelize,
+    modelName: 'User',
+  });
+  return User;
+};
```
Comment (ahaly92) [inline backend/src/db/models/user.js]: new line

File: frontend/src/index.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
+import { gql } from '@apollo/client';
+import store from './store/store';
+import configureFakeBackend from './utils/fake-backend';
+import history from './utils/history';
+
+import { productsActions } from './store/products/action';
+
+const client = new ApolloClient({
+  uri: 'http://localhost:4000/graphql/',
```
Comment (ahaly92) [inline frontend/src/index.js]: this should come from the env file

**Comment by ahaly92** [comment]
There's a .DS_Store checked in here

File: frontend/src/store/product/action.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+            product,
+          });
+        },
+        (error) => {
+          alert(error);
+          dispatch({ type: actionTypes.PRODUCT_FAILURE, error });
+        }
+      )
+      .then(() => {
+        const updatedProducts = JSON.parse(localStorage.getItem('products'));
```
Comment (ahaly92) [inline frontend/src/store/product/action.js]: this should be updated to get products from the backend. Check in with Spilman on this

File: frontend/src/utils/routes.js (hunk 2 above)
Comment (ahaly92) [reply frontend/src/utils/routes.js:1]: nice!

**Comment by ahaly92** [comment]
move the readme in front end to the root of this repo

**Pull request merged**